packages/widgets/src/accordionpanel.ts

```diff
@@ -53,6 +54,13 @@ export class AccordionPanel extends SplitPanel {
     return (this.layout as AccordionLayout).titles;
   }
 
+  /**
+   * A signal emitted when an element of the AccordionPanel is collapsed or expanded.
+   */
+  get expansionToggled(): ISignal<this, void> {
```

Comment:
Suggestion linked to the last minute thought (see further):

```suggestion
  get expansionToggled(): ISignal<this, number> {
```